dll/win32/getuname/lang/ro-RO.rc

```diff
@@ -24778,1696 +24779,2928 @@ BEGIN
+    65063 "Îmbinatorul jumătății din stânga a ligaturii de mai jos"
+    65064 "Îmbinatorul jumătății din dreapta a ligaturii de mai jos"
+    65065 "Îmbinatorul jumătății din stânga a tildăI de mai jos"
+    65066 "Îmbinatorul jumătății din dreapta a tildăI de mai jos"
+    65067 "Îmbinatorul jumătății din stânga a macronului de mai jos"
+    65068 "Îmbinatorul jumătății din dreapta a macronului de mai jos"
+    65069 "Îmbinatorul unirii macronului de mai jos"
+    65070 "Îmbinatorul titlo-ului chirilic cu jumătatea stângă"
+    65071 "Îmbinatorul titlo-ului chirilic cu jumătatea dreaptă"
+    65072 "Formular de prezentare pentru articol de fond vertical cu două puncte"
+    65073 "Formular de prezentare pentru linie lungă verticală"
+    65074 "Formular de prezentare pentru linie scurtă verticală"
+    65075 "Formular de prezentare pentru linia joasă verticală"
+    65076 "Formular de prezentare pentru linia joasă ondulată verticală"
+    65077 "Formular de prezentare pentru paranteza verticală din stânga"
+    65078 "Formular de prezentare pentru paranteza verticală din dreapta"
+    65079 "Formular de prezentare pentru paranteza ondulată verticală din stânga"
+    65080 "Formular de prezentare pentru paranteza ondulată verticală din dreapta"
+    65081 "Formular de prezentare pentru suportul vertical stâng al parantezei carapace de țestoasă"
+    65082 "Formular de prezentare pentru suportul vertical drept al parantezei carapace de țestoasă"
+    65083 "Formular de prezentare pentru paranteza lenticulară neagră verticală stângă"
+    65084 "Formular de prezentare pentru paranteza lenticulară neagră verticală dreaptă"
+    65085 "Formular de prezentare pentru unghiul vertical din stânga al parantezei duble"
+    65086 "Formular de prezentare pentru unghiul vertical din dreapta al parantezei duble"
+    65087 "Formular de prezentare pentru unghiul vertical din stânga al parantezei"
+    65088 "Formular de prezentare pentru unghiul vertical din dreapta al parantezei"
+    65089 "Formular de prezentare pentru colțul vertical din stânga al parantezei"
+    65090 "Formular de prezentare pentru colțul vertical din dreapta al parantezei"
+    65091 "Formular de prezentare pentru colțul vertical alb din stânga al parantezei"
+    65092 "Formular de prezentare pentru colțul vertical alb din dreapta al parantezei"
+    65093 "Punct de susan"
+    65094 "Punct alb de susan"
+    65095 "Formular de prezentare pentru paranteza pătrată verticală din stânga"
+    65096 "Formular de prezentare pentru paranteza pătrată verticală din dreapta"
+    65097 "Linie superioară întreruptă"
+    65098 "Linie centrală superioară cu punct în mijloc"
+    65099 "Linie de deasupra ondulată"
+    65100 "Linie de deasupra de două ori ondulată"
+    65101 "Linie inferioară întreruptă"
+    65102 "Linie centrală inferioară cu punct în mijloc"
+    65103 "Linie joasă ondulată"
+    65104 "Virgulă mică"
+    65105 "Virgulă ideografică mică"
```

Comment:
For this and the following lines, you may want to have the adjective translation for "small" placed immediately after the noun (like you did for resource №65129). Otherwise, it gets harder to make sense of what is it about, especially when one has to read things like "Semn de mai mare ca mic" (resource №65125).